src/Raven.Server/Documents/NotificationsClientConnection.cs

```diff
@@ -151,6 +151,9 @@ public void UnwatchAllTransformers()
 
         public void SendDocumentChanges(DocumentChangeNotification notification)
         {
+            // this is a precaution, in order to overcome an observed race condition between notification client disconnection and raising notifications
+            if (IsDisposed)
+                return;
```

Comment:
same here, we need to actually handle the failure of writing to the notification